libs/payments/management/project.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "payments-management",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/payments/management/src",
+  "projectType": "library",
+  "tags": ["scope:shared:lib:payments"],
+  "targets": {
+    "build": {
+      "executor": "@nx/esbuild:esbuild",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "main": "libs/payments/management/src/index.ts",
+        "outputPath": "dist/libs/payments/management",
+        "outputFileName": "main.js",
+        "tsConfig": "libs/payments/management/tsconfig.lib.json",
+        "assets": [
+          {
+            "glob": "libs/payments/management/README.md",
+            "input": ".",
+            "output": "."
+          }
+        ],
+        "platform": "node"
```

Comment:
Add `"declaration" : true` option to "build"

Ref: https://mozilla.github.io/ecosystem-platform/reference/npm-scripts-and-nx#creating-a-library-in-nx